src/panel/details.tsx

```diff
@@ -81,28 +105,49 @@ interface DetailsProps { result: Result, height: IObservableValue<number> }
                     </div>
                 </Tab>
                 <Tab name="Code Flows" count={this.threadFlowLocations?.length || 0}>
-                    <div className="svDetailsBody svDetailsCodeflow">
+                    <div className="svDetailsBody svDetailsCodeflowAndStacks">
                         {(() => {
                             const items = this.threadFlowLocations;
 
-                            const selection = observable.box<ThreadFlowLocation | undefined>(undefined, { deep: false });
+                            const selection = observable.box(undefined as Location, { deep: false })
                             selection.observe(change => {
-                                const tfloc = change.newValue;
-                                postSelectArtifact(result, tfloc?.location?.physicalLocation);
-                            });
-
-                            const renderItem = (tfLocation: ThreadFlowLocation) => {
-                                const { message, uri, region } = parseLocation(result, tfLocation.location);
-                                return <>
-                                    <div className="ellipsis">{message ?? '—'}</div>
-                                    <div className="svSecondary">{uri?.file ?? '—'}</div>
-                                    <div className="svLineNum">{region?.startLine ?? '—'}:1</div>
-                                </>;
-                            };
+                                const location = change.newValue
+                                postSelectArtifact(result, location?.physicalLocation)
+                            })
 
                             return <List items={items} renderItem={renderItem} selection={selection} allowClear>
```

Comment:
>renderItem [](start = 67, length = 10)

let's change this to renderLocation please

Reply:
Fixed